### PR TITLE
bootloader_start: Show which module's code is used

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/66114

https://openqa.suse.de/t4384296
https://openqa.suse.de/t4384297
https://openqa.suse.de/t4384298
https://openqa.suse.de/t4384300
https://openqa.suse.de/t4384301
https://openqa.suse.de/t4384335

### CODE REVIEW

File: tests/installation/bootloader_start.pm [hunk 1]
```diff
@@ -32,46 +32,62 @@ BEGIN {
...
     }
     # Load regular bootloader for all qemu backends and for x84_86 systems,
     # except Xen PV as id does not have VNC (bsc#961638).
     if (check_var('BACKEND', 'qemu') || (check_var('BACKEND', 'svirt') && !(check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux')))) {
         if (get_var('UEFI')) {
-            $self->bootloader_uefi::run();
-            return;
+            if (get_var('BOOT_HDD_IMAGE')) {
+                record_info('Starting uefi_bootmenu');
+                #$self->uefi_bootmenu::run();
```
Comment (SergioAtSUSE) [inline tests/installation/bootloader_start.pm:73]: Leftover?

Comment (asdil12) [reply tests/installation/bootloader_start.pm:73]: No - just commented out  as I'm debugging

File: tests/installation/bootloader_start.pm [hunk 2]
```diff
@@ -32,46 +32,62 @@ BEGIN {
     unshift @INC, dirname(__FILE__) . '/../boot';
 }
 use boot_from_pxe;
+use uefi_bootmenu;
 
 sub run {
     my $self = shift;
     if (uses_qa_net_hardware() || get_var("PXEBOOT")) {
+        record_info('Starting boot_from_pxe');
```
Comment (foursixnine) [inline tests/installation/bootloader_start.pm:40]: I'd go for 
```suggestion
        record_info('boot_from_pxe');
```

and the same for the rest of the bootloaders, ast the title in the record info box is smaller

File: tests/boot/uefi_bootmenu.pm [hunk 3]
```diff
@@ -16,13 +16,17 @@
...
 
+package uefi_bootmenu;
 use base "basetest";
 use strict;
 use warnings;
 use testapi;
 use utils;
 use bootloader_setup;
 
+require Exporter;
```
Comment (foursixnine) [inline tests/boot/uefi_bootmenu.pm:27]: this still haunts me a bit, but oh well.. :/

File: tests/installation/bootloader_start.pm (hunk 2 above)
Comment (asdil12) [reply tests/installation/bootloader_start.pm]: done

**Pull request merged**